services-core/src/metrics/stablex_metrics.rs

```diff
@@ -63,13 +64,22 @@ impl StableXMetrics {
         BookType::initialize_gauges(&users);
         registry.register(Box::new(users.clone())).unwrap();
 
+        let min_avg_fee_opts = Opts::new(
+            "dfusion_service_min_avg_fee",
+            "Computed min avg earned fee per order. This is the *earned* fee. It is doubled \
+             when passed to the solve which expects to receive the total *generated* fee.",
```

Comment:
Not sure if this value, or the doubled value is more natural. What do you think?

Reply:
Personally, I would prefer the value that is passed to the solver. This way it makes it easier for the solver team to diagnose issues with batches not finding trades.

Reply:
Changed it to the total generated fee, as passed to the solver.